src/features/VisualNovel/pages/CharactersPage/CharactersPage.tsx

```diff
@@ -59,6 +60,24 @@ export const CharactersPage: VFC = () => {
     return groupedCharacters;
   };
 
+  /** Calculates character column count. */
+  const getCharactersColumnCount = (): number => {
+    const [canFit4OrMore, canFitOnly3, canFitOnly2] = useMediaQuery(
```

Comment:
I am still thinking that names like `isScreenWidthLargerThan1024px` would be more semantically correct and reusable.